packages/osd-opensearch/README.md

```diff
@@ -9,9 +9,9 @@ To run, go to the OpenSearch Dashboards root and run `node scripts/opensearch --
 
 ### Examples
 
-Run a snapshot install with a trial license
+Run a snapshot install with an oss license
 ```
-node scripts/opensearch snapshot --license=trial
+node scripts/opensearch snapshot --license=oss
```

Comment:
Are these even valid we should remove it.

Reply:
will do